cli/flags_test.go

```diff
@@ -56,7 +56,7 @@ func TestNoLinkTesting(t *testing.T) {
 		}
 
 		for _, imp := range pkg.Imports {
-			// https: //github.com/golang/tools/blob/master/refactor/importgraph/graph.go#L115
+			// https://github.com/golang/tools/blob/master/refactor/importgraph/graph.go#L115
```

Comment:
Let me do that after this or #2688 lands so I don't have to keep updating both PRs